tools/restatectl/src/commands/log/list_logs.rs

```diff
@@ -44,24 +44,27 @@ pub async fn list_logs(connection: &ConnectionInfo, _opts: &ListLogsOpts) -> any
     )?;
 
     // sort by log-id for display
-    let logs: BTreeMap<LogId, &Chain> = logs.iter().map(|(id, chain)| (*id, chain)).collect();
+    let log_chains: BTreeMap<LogId, &Chain> = logs.iter().map(|(id, chain)| (*id, chain)).collect();
 
-    if logs.is_empty() {
+    if log_chains.is_empty() {
         c_println!(
-            "No logs found. Has the cluster been provisioned yet? You can do so with `restatectl provision`."
+            "No logs were found. Check if the cluster has been provisioned and partition processors have started on a worker node. `restatectl provision`."
         );
+        c_println!();
+        c_println!("Use `restatectl provision` if you did not provision this cluster yet.");
```

Comment:
Grammar nitpick - this should be present perfect tense:

```suggestion
        c_println!("Use `restatectl provision` if you have not provision this cluster yet.");
```